.github/workflows/blackduck-scan.yml

```diff
@@ -0,0 +1,112 @@
+###
+##  Foundation-security BlackDuck workflow for public repos
+#
+name: Foundation-Security/Black Duck Scan (PUBLIC)
+
+on:
+  push:  
+    tags:        
+      - '**'          
+  workflow_dispatch:
+
+jobs:
+  Blackduck-Scan:
+    runs-on: ubuntu-20.04
+    steps:
+
+    - name: Checkout source repository
+      id: checkout-source 
+      uses: actions/checkout@v3
+
+    - name: Download and Run Detect
+      id: run-synopsys-detect
+      env:
+        detect.blackduck.scan.mode: 'INTELLIGENT'
+        detect.scan.output.path: '/home/runner/work/_temp/blackduck'
+        detect.output.path: '/home/runner/work/_temp/blackduck'
+        detect.project.version.name: ${{ github.ref_type }}/${{ github.ref_name }}
+        detect.project.name: ${{ github.event.repository.name }}
+        detect.risk.report.pdf: true
+        detect.risk.report.pdf.path: /tmp/reports/
+        blackduck.url: ${{ secrets.BLACKDUCK_URL }}
```

Comment:
This will cause a failure if anyone with a fork does not have the relevant secrets configured. This needs to be guarded with an appropriate check to ensure the secrets are present.

Reply:
`if: secrets.BLACKDUCK_URL != null` has been added. Evidentially it will cancel out the Deploy-to-S3 job too, as the Blackduck-Scan is needed.